crates/steel/src/host/builder.rs

```diff
@@ -224,11 +234,39 @@ impl<P, H> EvmEnvBuilder<P, H, ()> {
 #[stability::unstable(feature = "history")]
 #[derive(Clone, Debug)]
 pub struct History {
-    beacon_url: Url,
+    beacon_config: Beacon,
     commitment_block: BlockId,
 }
 
-impl<P> EvmEnvBuilder<P, EthBlockHeader, Url> {
+impl<P> EvmEnvBuilder<P, EthBlockHeader, Beacon> {
+    /// Creates a consensus commitment instead of a beacon commitment that can be verified using the
+    /// EIP-4788 beacon root contract.
```

Comment:
This sentence is a bit unclear which of the two, consensus or beacon, can be verified using the EIP-4788 beacon root contract. What about the following?
```suggestion
    /// Creates a consensus commitment, with a slot number as the identifier.
```

Reply:
addressed in d2f9b9a3615b9d0adcd42a8eb537e759db6321a2